File: 0231-power-of-two/0231-power-of-two.c
#define TRUE true
#define FALSE false

bool isPowerOfTwo(int n) {
    if (n < 1) {
        return FALSE;
    } else if (n == 1) {
        return TRUE;
    } else {
        while (n % 2 == 0) {
            n /= 2;
        }
        if (n == 1) {
            return TRUE;
        }
        return FALSE;
    }
}

File: 0231-power-of-two/0231-power-of-two.java
class Solution {
    public boolean isPowerOfTwo(int n) {
        if(n < 1) {
            return false;
        } else if( n == 1) {
            return true;
        } else {
            if(n % 2 == 0) {

                while(n % 2 == 0) {
                    n /= 2;
                }

                if( n == 1) {
                    return true;
                }

                return false;
            }

            return false;
        }
    }
}